Pocket Casts Watch App Extension/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we move this file to `Pocket Casts Watch App` folder instead? This extension folder is going to be removed soon in this [PR](#1516 ) so this will make it easier to transition.

Reply:
Done!